### DISCUSSION TITLE
Victory Chart with multiple y-axis having independent domains

### DISCUSSION THREAD

**Discussion by vikasraj789** [FormidableLabs/victory#2556] [Q&A]
We are displaying Victory chart with VictoryLine using two dependant VictoryAxis. Currently, both axis are getting same tick labels, even if we send different domains to each. We want to display two dependant axis with different tick values. Please help.

**Comment by vikasraj789** [comment]
@boygirl Can you pls help solve the issue?

**Comment by pechea** [comment]
From the Victory chart FAQ, it appears that the library does not support multiple y-axes with different domains. However, there are workarounds: https://formidable.com/open-source/victory/docs/faq/#can-i-make-a-chart-with-multiple-dependent-axes.
